library/firewall_lib_facts.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016,2017,2020,2021 Red Hat, Inc.
+# Reusing some firewalld code
+# Authors:
+# Brennan Paciorek <bpaciore@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Does this heading look correct? It feels incorrect to list myself as the sole author here

Reply:
Looks good to me.

Reply:
There is no problem listing yourself as sole author.